architecture/routing.adoc

```diff
@@ -107,24 +107,53 @@ file.
 
 ====
 
-IMPORTANT: While both secure and unsecure routes are possible with an OpenShift
-instance, Red Hat recommends a secure routing service.
+IMPORTANT: While both secure and unsecure routes are possible with an OpenShift instance, Red Hat recommends a secure routing service.
 
```

Comment:
@bfallonf We need to be careful when we document things such as "Red Hat recommends....." considering that the content also applies to the Origin project, which is open source. I'm not sure how others would feel about it, but I think Red Hat should be specific to the RH Enterprise products - OSE and Online. So I think here you need to conditionalize this text for Origin/Online/Enterprise.